modules/sql-engine/src/integrationTest/sql/types/blob/test_blob.test

```diff
@@ -87,3 +88,55 @@ SELECT CAST(x'0101' AS BINARY(0))
 statement error: Length for type VARBINARY must be at least 1
 SELECT CAST(x'0101' AS VARBINARY(0))
 
+statement ok
+CREATE TABLE all_blobs (c1 binary, c2 binary(10), c3 varbinary, c4 varbinary(10), c5 binary varying, c6 binary varying(10));
+
+statement ok
+INSERT INTO all_blobs VALUES (x'cc', x'ccaaffee', x'ccaaffee', x'ccaaffee', x'ccaaffee', x'ccaaffee');
+
+query TTTTTT
+SELECT * FROM all_blobs;
+----
+cc	ccaaffee	ccaaffee	ccaaffee	ccaaffee	ccaaffee
+
+statement ok
+DELETE FROM all_blobs
+
+skipif ignite3
+# https://issues.apache.org/jira/browse/IGNITE-22316
+# check standard section: <binary value expression> ::= <binary concatenation> | <binary factor>
+query T
+SELECT x'12' || '10';
+----
+42
+
+query T
+select X'' || X'12'
+----
+12
+
+query T
+select X'11' || X'22';
+----
+1122
+
+query T
+select CAST(x'Aa' AS BINARY VARYING(1000));
+----
+aa
+
+#sql standard, 6.30 <numeric value function> If <binary position expression> is specified
+query T
+SELECT POSITION(x'03' IN x'0B03');
+----
+2
+
+query T
+SELECT POSITION(x'b0' IN x'0B03B0');
+----
+3
+
+query T
+SELECT POSITION(x'b0' IN x'0B03B0B0');
+----
+3
```

Comment:
Lets move length/octet_length test from `ItFunctionsTest` to SQL logic tests.

I see POSITION function tests for strings in sql/function/string/test_position.test.
Maybe we should go similar way and move function's tests into sql/function/blob

Reply:
position - done into new file,
octet_length - the only thing i can do here - is to remove 1 line with **octet_length** from ItFunctionsTest cause this one already tests in test_blob_function.test. But seems we need to talk here i think that to have some duplicates in It* tests and SqlLogic*tests it`s ok just because It* tests can be run locally and passed fast while SqlLogic are very time consumable